.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,338 @@
+name: integration-tests
+on:
+  push:
+    branches: [ "518-implement-semi-automatic-integration-tests", "642-integration-tests-add-tests-for-aliens", "648-integrationtests-add-consistency-test-for-random-putgetdelete-requests-with-bobt-tool" ] #REMOVE WHEN ON MASTER
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+(.alpha.[0-9]+)*"
+  pull_request_review:
+    types: [submitted]
+  workflow_dispatch:
+    inputs:
+      bob_nodes_amount:
+        description: 'Amount of nodes in bob cluster for tests'
+        required: true
+        default: '4'
+      bob_vdisk_count:
+        description: 'Vdisks count'
+        required: true
+        default: '4'
+      bob_replicas_amount:
+        description: 'Amount of vdisks replicas'
+        required: true
+        default: '2'
+      bob_payload_size:
+        description: 'Payload size'
+        required: true
+        default: '10240'
+      bob_record_count:
+        description: 'Amount of records to process during test'
+        required: true
+        default: '100000'
+      bob_first_index:
+        description: 'Index of the first record to proceed during tests'
+        required: true
+        default: '0'
+      bob_bobt_count:
+        description: 'Amount of iterations for chain operation test'
+        required: true
+        default: '100000'
+      bob_bobt_first_index:
+        description: 'Bottom boundry for chain test'
+        required: true
+        default: '0'
+      bob_bobt_last_index:
+        description: 'Upper boundry for chain test'
+        required: true
+        default: '10000'
+jobs:
+  build-binaries:
+    if: ${{ github.event_name != 'pull_request_review' || github.event.review.state == 'approved' }}
+    strategy:
+      matrix:
+        key: [8, 16]
+    env:
+      TARGET: "x86_64-unknown-linux-musl"
+      BUILD_MODE: "debug"
```

Comment:
```suggestion
      BUILD_MODE: "test"
```

Reply:
About Rust build profiles (https://doc.rust-lang.org/cargo/reference/profiles.html#default-profiles):
Rust has 4 default build profiles: `dev`, `release`, `test`, `bench`.
Additionally, Rust allows to declare custom profiles, but at the moment we don't have those.
Best profile for integration tests is `test` (it even has the name, indicating that).

There is no profile with name `debug` in Rust. Instead it has `dev` profile, that essentially the same.
That is the reason behind `debug` -> `dev` transformation.

Additionally, profile name affect the output directory for binaries. For all profiles, except `dev` the output directory equals to the profile name. But for `dev` profile, unfortunately, the directory is called `debug`. 
Example:
1. `release` -> `./target/x86_64-unknown-linux-musl/release/`
2. `test` -> `./target/x86_64-unknown-linux-musl/test`
3. `dev` -> `./target/x86_64-unknown-linux-musl/debug`